workflows/process/checkV/checkV.nf

```diff
@@ -1,24 +1,25 @@
 process checkV {
-        publishDir "${params.output}/${name}/${category}", mode: 'copy'
```

Comment:
would favor params output more. easier and more transparent to change (in a dedicated params section). not a friend of hardcoded output dirs